invenio_vocabularies/services/service.py

```diff
@@ -196,3 +205,21 @@ def read_many(self, identity, type, ids, fields=None, **kwargs):
             extra_filter=filter, **kwargs)
 
         return self.result_list(self, identity, results)
+
+    def launch(self, identity, data):
+        """Create a task.
+
+        FIXME: This is a PoC. The final implementation should resemble
+        the PIDs, having a sub-service and a manager. If persistance is
+        added UoW should be used.
+        """
+        self.require_permission(identity, "manage")
+        task_config, _ = self.task_schema.load(
+            data,
+            context={"identity": identity},  # FIXME: is this needed
+            raise_errors=True
+        )
+        launch_async_vocab_task(task_config)
```

Comment:
```suggestion
        launch_async_vocab_task.delay(task_config)
```

To call it async. But maybe there are troubles with passing the `Identity` object in there... I guess it goes back to the discussion we had about how to "serialize" an identity.